docs/src/apps/jsonrpc-api.md

```diff
@@ -332,6 +332,7 @@ The result field will be an object with the following fields:
       - `fee: <u64>` - fee this transaction was charged, as u64 integer
       - `preBalances: <array>` - array of u64 account balances from before the transaction was processed
       - `postBalances: <array>` - array of u64 account balances after the transaction was processed
+      - `invoked: <array|undefined>` - List of [invoked instructions](#invoked-instructions-structure) or omitted if invoked instruction recording was not yet enabled during this transaction
```

Comment:
To me `invoked` alone doesn't mean too much.  WDYT about `invokedInstructions`?   Although normal instructions are invoked too.  `innerInstructions`?

Reply:
Ah, I like inner!